### ISSUE TITLE
Fixed toolbar not working for nested Vizy fields

### ISSUE THREAD

**Issue by adrienpicard** [verbb/vizy#23]
**Description**
The`toolbarFixed` setting only works for the top level Vizy editor. It doesn't for nested ones.

**Steps to reproduce**
1. Create a Vizy field
2. Create a block with another Vizy field, set `toolbarFixed: true`
3. Create an Entry and test it out 

**Additional info**
- Plugin version: 1.0.1
- Craft version: 3.6
- Multi-site: No

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by adrienpicard** [comment]
Thank you!

**Comment by engram-design** [comment]
Fixed in [1.0.2](https://github.com/verbb/vizy/releases/tag/1.0.2)
